packages/clerk-js/src/ui/components/SessionTasks/tasks/TaskSelectOrganization/CreateOrganizationScreen.tsx

```diff
@@ -0,0 +1,170 @@
+import { useSessionTasksContext } from '@/ui/contexts/components/SessionTasks';
+import { Col, descriptors, Flex, Icon, localizationKeys } from '@/ui/customizables';
+import { useCardState, withCardStateProvider } from '@/ui/elements/contexts';
+import { Form } from '@/ui/elements/Form';
+import { FormButtonContainer } from '@/ui/elements/FormButtons';
+import { FormContainer } from '@/ui/elements/FormContainer';
+import { IconButton } from '@/ui/elements/IconButton';
+import { Organization } from '@/ui/icons';
+import { createSlug } from '@/ui/utils/createSlug';
+import { handleError } from '@/ui/utils/errorHandler';
+import { useFormControl } from '@/ui/utils/useFormControl';
+
+import { organizationListParams } from '../../../OrganizationSwitcher/utils';
+
+type CreateOrganizationScreenProps = {
+  onCancel?: () => void;
+};
+
+export const CreateOrganizationScreen = withCardStateProvider((props: CreateOrganizationScreenProps) => {
+  const card = useCardState();
+  const { __internal_navigateToTaskIfAvailable } = useClerk();
+  const { redirectUrlComplete } = useSessionTasksContext();
+  const { createOrganization, isLoaded, setActive } = useOrganizationList({
+    userMemberships: organizationListParams.userMemberships,
+  });
+
+  const [file, setFile] = useState<File | null>();
+  const nameField = useFormControl('name', '', {
+    type: 'text',
+    label: localizationKeys('formFieldLabel__organizationName'),
+    placeholder: localizationKeys('formFieldInputPlaceholder__organizationName'),
+  });
+  const slugField = useFormControl('slug', '', {
+    type: 'text',
+    label: localizationKeys('formFieldLabel__organizationSlug'),
+    placeholder: localizationKeys('formFieldInputPlaceholder__organizationSlug'),
+  });
+
+  const onSubmit = async (e: React.FormEvent) => {
+    e.preventDefault();
+
+    if (!isLoaded) {
+      return;
+    }
+
+    try {
+      const organization = await createOrganization({ name: nameField.value, slug: slugField.value });
+
+      if (file) {
+        await organization.setLogo({ file });
+      }
+
+      await setActive({ organization });
+
+      await __internal_navigateToTaskIfAvailable({ redirectUrlComplete });
+    } catch (err) {
+      handleError(err, [nameField, slugField], card.setError);
+    }
+  };
+
+  const onAvatarRemove = () => {
+    card.setIdle();
+    return setFile(null);
+  };
+
+  const onChangeName = (event: React.ChangeEvent<HTMLInputElement>) => {
+    nameField.setValue(event.target.value);
+    updateSlugField(createSlug(event.target.value));
+  };
+
+  const updateSlugField = (val: string) => {
+    slugField.setValue(val);
+  };
+
+  const isSubmitButtonDisabled = !nameField.value || !isLoaded;
+
+  return (
+    <FormContainer>
+      <Form.Root
+        onSubmit={onSubmit}
+        sx={t => ({ padding: `${t.space.$none} ${t.space.$10} ${t.space.$8}`, gap: t.space.$8 })}
+      >
+        <Flex
+          direction='col'
+          gap={6}
+        >
+          <Col>
```

Comment:
Do we need these extra Flex and Cols in use here? Form.Root is a Col with gap={6} so it looks like we get the same output.

Reply:
There's a spacing of 32 pixels between the form container and the header, so we added gap on the form container. 

I've just checked tho and it wasn't making an effect. Removed the `Col`, left the default `gap` on the form container, but added a margin top to sum up 32px.